crates/cxx-qt-gen/test_outputs/properties.rs

```diff
@@ -113,17 +113,23 @@ mod cxx_qt_ffi {
     }
 
     impl MyObject {
-        pub fn get_primitive<'a>(&'a self, cpp: &'a MyObjectQt) -> &'a i32 {
-            cpp.get_primitive()
+        pub fn primitive<'a>(&'a self, cpp: &'a MyObjectQt) -> &'a i32 {
+            cpp.primitive()
         }
     }
 
     impl MyObjectQt {
-        pub fn get_primitive(&self) -> &i32 {
+        pub fn primitive(&self) -> &i32 {
             &self.rust().primitive
         }
     }
 
+    impl MyObjectQt {
+        pub unsafe fn primitive_mut<'a>(mut self: Pin<&'a mut Self>) -> Pin<&'a mut i32> {
```

Comment:
Should we return `&mut T` or `Pin<&mut T>` here?

Reply:
For Rust-native types, probably a `&mut T`.
If you want to pin the type yourself, you can use a `Pin<Box<T>>` or similar

Reply:
OK, done :-)